src/turnkeyml/build/stage.py

```diff
@@ -280,7 +280,7 @@ def launch(self, state: build.State) -> build.State:
         # Collect telemetry for the build
         stats = fs.Stats(state.cache_dir, state.config.build_name, state.evaluation_id)
         stats.save_model_eval_stat(
-            fs.Keys.ALL_BUILD_STAGES,
+            fs.Keys.SELECTED_SEQUENCE_OF_STAGES,
```

Comment:
I'm not sure about this name because its introducing new terminology, `selected`, that we aren't using anywhere else. Technically the user selects all the tkml CLI options, but we don't point that out in any other stat.

How often do we think this will be queried? Since the `stage_status` stats will now be comprehensive...

If we think this wont be queried too often, I have a pretty different proposal (that solves some other problems for us as well): capture the entire turnkey CLI command for the build into a dict.

It would look like this:

```yaml
- turnkey_options:
  - seqeuence: [Stage, Stage, Stage]
  - onnx_opset: 14
  - timeout: 3600
  - etc.
```

We need to do the above because otherwise we are constantly playing whack a mole trying to capture all of the info we need. Better to automatically capture it all in one shot.

So I am wondering if the above is also a replacement for capturing `SELECTED_SEQUENCE_OF_STAGES` as its own independent stat? If you agree, I suggest just removing the stat entirely in this PR, and I can create an issue to capture the TKML command as a stat in a future PR.

Reply:
> How often do we think this will be queried? Since the stage_status stats will now be comprehensive...
 The order of  the stages in the columns of the report (stage_status) can't be guaranteed, as different builds may have a different order of stages. in other words, `SELECTED_SEQUENCE_OF_STAGES` is still needed to understand the order of those stages. Therefore, I still see a value of `SELECTED_SEQUENCE_OF_STAGES` being its own independent stat.